newIDE/app/src/EventsFunctionsList/index.js

```diff
@@ -323,6 +337,7 @@ export default class EventsFunctionsList extends React.Component<Props, State> {
                       i18n
                     )}
                     reactDndType="GD_EVENTS_FUNCTION"
+                    isItemBold={getEventsFunctionPrivateness}
```

Comment:
Do you think we could just add ` (private)` at the end of the name shown in the list?

Reply:
Personally, I prefer the boldness, but if you think this is better then yes no problem